### PR TITLE
Std norm cdf test

### PR DESCRIPTION
**Pull request by Chepelau**

### CODE REVIEW

**Comment by MarekHauzr** [comment]
which one of the two contracts returns error around x=7?

**Comment by MarekHauzr** [comment]
@Chepelau have you had a look at relative differences? the abs differences between something close to 0 and something close to 0 for different "something close to 0" will be basically the same

**Comment by MarekHauzr** [comment]
@Chepelau the python approximation with error function solves the problem that you were mentioning that for x>10 and x <-10 the python implementations return 0 and 1?

File: examples/comparing_std_norm_cdfs/bs.cairo [hunk 1]
```diff
@@ -0,0 +1,565 @@
+# Declare this file as a StarkNet contract and set the required
```
Comment (MarekHauzr) [inline examples/comparing_std_norm_cdfs/bs.cairo:1]: to this and the other contract... please add a source of what it is and where it comes from

**Comment by MarekHauzr** [comment]
@Chepelau the notebook should have a headline and a description

**Comment by MarekHauzr** [comment]
@Chepelau just to make sure, that I understand it correctly
- our implementation has the highest abs error, that happens around +/- 2
- `Cairo Implementations vs SciPy (in %)` - why does it fall so suddently to -1? I guess because the cairo values drop to zero so fast?
- `Cairo Implementations vs SciPy (in %)` - this would suggest that our implementation is the best from perspective of the range of usable values (cca -8 to 8... second and third ~-5 to 5)
- at the end of the notebook `er, other contracts will return 0 or 1 sooner the ours.` ... `the` -> `than`?

otherwise good job :) thanks a lot

**Comment by Chepelau** [comment]
@MarekHauzr 
1) Yes, our implementation has the highest abs errors, at least visually around +/- 2.
2) Yes, other contracts converge to 0/1 due to the fact that it is hardcoded( they implement MAX/MIN_CDF_INPUT), our probably due to the approximation not being able to achieve precision when it comes to such  outliers. 
3) Indeed. 
4) Fixed.

**Comment by MarekHauzr** [comment]
@Chepelau thanks

**Pull request merged**